data-raw/healthy-places/access-to-green-space/access-greenspace.R

```diff
@@ -0,0 +1,25 @@
+# Load packages
+library(tidyverse)
+library(geographr)
+library(devtools)
+library(styler)
+
+# Load lookup dataset
+lookup_ltla_raw
```

Comment:
This line (and therefore subsequent lines) error because `lookup_ltla_raw` is not the name of the dataset from the geographr package. The names need to match the data file that is in the package.